### ISSUE TITLE
Where are the Go communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/go#1111]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Go supported by one or more large organizations?
* Does Go have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Go? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Go? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Go? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Go?
* Are there popular newsletters for Go?
* Is Go taught at programming bootcamps? (If so, what are some examples?)
* Is Go taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Go?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by exklamationmark** [comment]
Hi @kytrinyx, thanks for creating this. It will be a very useful index to all the learning resources!

An interesting course with Go is [MIT’s 6.824: Distributed system](https://pdos.csail.mit.edu/6.824/). Students will get to build MapReduce, Raft and a distributed key-value store using Go.

There are more courses as mentioned in [this wiki page](https://github.com/golang/go/wiki/Courses).

**Comment by sebito91** [comment]
I have a handle on a few sources, hopefully this helps...

1. [awesome-go](https://github.com/avelino/awesome-go/blob/master/README.md)
2. gophers.slack.com (created an exercism channel a while back, but stayed pretty dormant)
3. the discord golang [channel](https://discord.gg/tCv8sAn)

I've put the feelers out to anybody I know in the community so fingers crossed some others will sign up!
